jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants.properties

```diff
@@ -55,7 +55,7 @@ Clear=Clear
 Save=Save
 Variables_Name=Variable Name
 Variable_Value=Variable Value
-Signalling_Process_Instance=Signaling Process Instance (id={0})
+Signalling_Process_Instance=Signaling Process Instance
```

Comment:
@cristianonicolai maybe we can remove this one? and use Signaling_Process_Instance

Reply:
Sure, will send a new PR for it